lib/imports/api/prosemeteor/server/prosemeteor-server.js

```diff
@@ -1,31 +1,46 @@
 import { Meteor } from 'meteor/meteor';
 import { AuthorityManager } from './../../authority/server/authority-manager';
 import { documentsColl } from './../../documents/both/collection';
-
+import { proseMeteorConfig } from './../../config/server/prosemeteor-config';
 /**
 * Main ProseMeteor server class that instantiates all of ProseMeteor on the server.
 */
 export class ProseMeteorServer {
   /*
   * Main ProseMeteor class on the server.
   * @param {Object} params
-  * @param {Collection} params.documentsColl     Collection PM docs are stored in
+  * @param {String} params.protocol              protocol this server uses ('http' or 'https')
   * @param {Number} [params.snapshotIntervalMs]  interval in milliseconds that documents are backed up to a snapshot in db
   * @constructor
   */
-  constructor ({ snapshotIntervalMs = 5000 }) {
+  constructor ({
+    // set default config values if not provided.
```

Comment:
all default config values are defined here in the `ProseMeteorServer` constructor